packages/labs/virtualizer/src/Virtualizer.ts

```diff
@@ -223,6 +223,13 @@ export class Virtualizer {
   private _layoutCompleteRejecter: Function | null = null;
   private _pendingLayoutComplete: number | null = null;
 
+  /**
+   * Layout initialization is async because we dynamically load
```

Comment:
How big is the default layout? Could we just statically import it?

Reply:
Can dig into the size numbers and reconsider the tradeoffs here, but the default `flow` layout is non-trivial in size, and previous thinking was that it was definitely worth not forcing folks to pay that cost if they were using an alternative layout.

Another option, which we actually briefly tried out at some point in the `0.x` history, is to always require a layout to be explicitly loaded and provided by the app. The DevEx hit for the 80% case was unfortunate, though, so we reverted to dynamically auto-loading.

At this point, with all of the async polyfill stuff out of the picture and some kinks worked out, I think we've converged on a dynamic loading scheme that isn't too heavy and shouldn't cause much trouble going forward, so my inclination is to keep this as-is for now.